stories/link_test.fmf

```diff
@@ -0,0 +1,25 @@
+/link_new_test:
+    story:
+        As a user I want to link a newly created test to a Jira issue.
+
+    description:
+        User should be able to link a newly created test/plan to JIRA issue.
+
+    example:
+      - |
+        command: tmt test create /tests/smoke --link verifies:https://foo.com/a/b/c --generate-url
```

Comment:
```suggestion
        tmt test create /tests/smoke --link verifies:https://foo.com/a/b/c
```

I believe the linking could happen by default. Let's try to avoid an extra option, hm?